data_science/specs/cohort_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rspec'
+require 'json'
+require_relative '../cohort.rb'
+
+RSpec.describe 'Cohort' do
+  let (:test_data) { JSON.parse(File.read('../test_data.json')) }
+  let (:cohort_a) { test_data.select { |v| v['cohort'] == 'A' } }
```

Comment:
It's incredibly expensive to initialize everything just for a subsection. The `Cohort` class doesn't care about JSON so we should pass in a flat object instead of digging around JSON.